task.py

```diff
@@ -46,10 +48,17 @@ def initialize_from_json(self, json_data):
       for dic in task_metrics[DISK_UTILIZATION_KEY]["Device Name To Utilization"]:
         for device_name, block_utilization in dic.iteritems():
           self.disk_utilization[device_name] = (
```

Comment:
Can you put these into an object instead of the 5-tuple?  It's getting big enough that I'm worried about potential for bugs, particularly with swapping the start and end counters.